packages/forms-web-app/src/sass/mobile/main.scss

```diff
@@ -0,0 +1,16 @@
+.mobile-show {
+	display: none;
+}
+
+.mobile-hide {
+}
+
+@media screen and (max-width: 640px) {
```

Comment:
Use GDS sass variables for breakpoints. https://github.com/alphagov/govuk-frontend/blob/v4.5.0/src/govuk/settings/_media-queries.scss#L23-L23